### PR TITLE
proper @jorgectf handle

### PR DESCRIPTION
**Pull request by felickz**
@jorge_ctf to @jorgectf

### CODE REVIEW

File: CVE-2022-22965/spring-rce.ql [hunk 1]
```diff
@@ -73,4 +73,4 @@ predicate vulnSpringJarFile(JarFile file) {
 from SpringRequestMappingMethod m
 where 
     (m.getAParameter().getType() instanceof SpringUntrustedDataType) and vulnSpringJarFile(_)
-select m, "Vulnerable use of Spring versions for CVE-2022-22965; Check if Java version >= 9 to confirm exploitability."
```
Comment (felickz) [inline CVE-2022-22965/spring-rce.ql:76]: GH.com added newline at EOF :)

File: CVE-2022-22965/spring-rce.ql [hunk 2]
```diff
@@ -9,7 +9,7 @@
  *       CVE-2022-22965
  * 
  * Lead Author: Alvaro Munoz (@pwntester)
- * Co-Author: Jorge (@jorge_ctf)
```
Comment (felickz) [inline CVE-2022-22965/spring-rce.ql:12]: handletypeo

**Comment by jorgectf** [comment]
Thanks! :D 

// cc: @securingdev

**Comment by securingdev** [comment]
Thanks @felickz! 

Sorry about that @jorgectf! I was initially looking at Twitter profiles and didn't realize they were different 🙇

**Pull request merged**